packages/react-router/modules/Router.js

```diff
@@ -23,7 +21,7 @@ class Router extends React.Component {
   getChildContext() {
     return {
       router: {
```

Comment:
No need to use the `router` key here. Let's just put everything directly on the `context` object.

Reply:
This is to maintain BC with those that are relying on our existing legacy Context usage.